aspnetcore/fundamentals/minimal-apis/10.0-samples/MinimalServerSentEvents/HeartRateRecord.cs

```diff
@@ -0,0 +1,4 @@
+public record HeartRateRecord(DateTime Timestamp, int HeartRate)
+{
+    public static HeartRateRecord Create(int heartRate) => new(DateTime.UtcNow, heartRate);
+}
```

Comment:
I changed the name from `HearRate` to `HeartRateRecord`
see original https://github.com/captainsafia/minapi-sse/blob/main/Program.cs#L60-L63